solhint.js

```diff
@@ -5,9 +5,12 @@ const _ = require('lodash')
 const fs = require('fs')
 const process = require('process')
 const linter = require('./lib/index')
+const packageJson = require('./package.json')
+const { FileNotExistsError } = require('./lib/common/errors')
 
 function init() {
-  program.version('1.1.10')
+  const version = packageJson.version || '1.1.10'
```

Comment:
I think we can remove the `1.1.10` fallback here.

Reply:
Solved